UI/Panels/Settings/MobiFlightPanel.cs

```diff
@@ -848,6 +848,7 @@ private void openToolStripButton_Click(object sender, EventArgs e)
 
                 moduleNode.ImageKey = "Changed";
                 moduleNode.SelectedImageKey = "Changed";
+                uploadToolStripButton.Enabled = true;
```

Comment:
there is nothing in the code that disables the button on opening inside this function. That's why I find the proposed solution doesn't fix the problem.

In your screen recoding the device is not selected by the user, but when you click the open button- i would guess that "afterselect"-event is not properly triggered in this case.

So the real fix should be to properly "select" the node and trigger the `mfModulesTreeView_AfterSelect` correctly.

Expected behavior:
* If no board is selected, the toolbar should be deactivated.
* If one board is selected, the toolbar items become active.
* If I open a new config, the board should stay selected as active, and all buttons in the toolbar therefore active too.